extensions/api/auth-tokenbased/src/main/java/org/eclipse/dataspaceconnector/api/auth/TokenBasedAuthenticationExtension.java

```diff
@@ -56,7 +55,7 @@ public void initialize(ServiceExtensionContext context) {
             apiKey = context.getSetting(AUTH_SETTING_APIKEY, UUID.randomUUID().toString());
         }
 
-        context.getMonitor().info(format("API Authentication: using static API Key '%s'", apiKey));
+        context.getMonitor().info("API Authentication: using static API Key");
```

Comment:
I would just override the `name()` method making it return like "Static token API Authentication"